MAINTAINERS

```diff
@@ -2,11 +2,15 @@ The maintainers are generally available in Slack at
 https://cloud-native.slack.com in #flux (https://cloud-native.slack.com/messages/CLAJ40HV3)
 (obtain an invitation at https://slack.cncf.io/).
 
-In additional to those listed below, this project shares maintainers
+In additional to those listed as current maintainers below, this project shares maintainers
 from the main Flux v2 git repository, as listed in
 
     https://github.com/fluxcd/flux2/blob/main/MAINTAINERS
 
-In alphabetical order:
+Project maintainers, in alphabetical order:
 
-Michael Bridgen, Weaveworks <michael@weave.works> (github: @squaremo, slack: Michael Bridgen)
+None.
```

Comment:
This looks like an issue in need of adjustment, :)

Is there anything in governance about this? (I suppose we'll need someone to volunteer, else there will be nobody for reviewing and merging PRs.)

Reply:
This is confusing indeed, it's not like this project is now abandoned and without maintainers, but having `None` here suggests that.

Reply:
@squaremo can you please remove the "Project maintainers, in alphabetical order:" section?

Reply:
The alternative is that the initial paragraph refers to a list that does not follow.

Reply:
If you don't read the initial paragraph, either "None" or a missing list will lead you to think there are no maintainers. If you do read it, you will understand that "None" refers to additional maintainers.

Reply:
In other repos we don't say `None` we just omit that paragraph: https://github.com/fluxcd/kustomize-controller/blob/main/MAINTAINERS

Reply:
Yep, that's a better formulation: no need to refer to additional maintainers, then not have any.